### PR TITLE
Books-with-classes

### PR DESCRIPTION
**Pull request by Nezerwa**
In this branch we :
-Used JavaScript classes instead of objects and arrays.
-Created class methods to add and remove books.
-Maintained all of the original functionality of the application
- Didn't use any library or framework, use only plain JavaScript.
-Added CSS styles to the application

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,58 +1,32 @@
...
-           <label for="Author"></label>
-           <input type="name" required id="author" name="author" placeholder="author" >
-          </div>
-        <div>
-          <button id="add-btn" type="submit">
-             Add
-          </button>
-          <span></span>
+<body>
+    <section class="main-container">
```
Comment (Lizdev-05) [inline index.html]: Great job implementing the functionalities 👍, According to the wireframe, you are to add a navigation bar. Kindly add it to meet up with the requirement. 

| Your Design | Wireframe Design |
| :---:   | :---: | 
| image|image|

Kindly refer back to the wireframe and add the necessary pages.

File: index.js [hunk 2]
```diff
@@ -1,93 +1,59 @@
...
+};
+
+// to add book from books collection
+const booksForm = document.querySelector('.new-book-form');
+booksForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const id = Math.floor(Math.random() * 10000); // Asign random Id for each
+  const item = new Book(id, title, author); // Add book for the class Book
```
Comment (Lizdev-05) [inline index.js]: Kindly check the add functionality, I tried adding a book and author but it's not adding. Kindly make the necessary changes as this is part of the core functionalities of your project.⚔️🟢

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @Nezerwa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- JavaScript classes were used ✅
- Linters are running ✅

## Required Changes ♻️

- Please, you are to have three content sections 
  -   Books list, 
  -   Add book form and
  -   Contact info
Kindly add the sections as we have them in the requirement.😊

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Yazino12 — APPROVED** [review]
## Hi @monicah79 and @Nezerwa 🖐🏾,

You've done well implementing the project requirements 🎉🎊🎉, and there is nothing else left to say other than it's time to merge it 🤝🏽

### To highlight:
- Linters are passing ✔️
- Correct use of GitHub flow ✔️ 
- Add and Remove functionalities are working ✔

# STATUS:  APPROVED ✅✅✅
THERE
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions

- **_[OPTIONAL]_** In your future collaborative projects, please keep in mind that **all** partners need to contribute to the project. This can be seen in your commits 📝.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**